Source/CCZ4/Weyl4.impl.hpp

```diff
@@ -132,51 +132,55 @@ EBFields_t<data_t> Weyl4::compute_EB_fields(
         }
     }
     // covariant derivative of K
-    FOR3(i, j, k) { covariant_deriv_K_tensor[i][j][k] = d1_K_tensor[i][j][k]; }
-
-    FOR4(i, j, k, l)
+    FOR3(i, j, k)
     {
-        covariant_deriv_K_tensor[i][j][k] +=
-            -chris_phys[l][k][i] * K_tensor[l][j] -
-            chris_phys[l][k][j] * K_tensor[i][l];
+        covariant_deriv_K_tensor[i][j][k] = d1_K_tensor[i][j][k];
+
+        FOR1(l)
+        {
+            covariant_deriv_K_tensor[i][j][k] +=
+                -chris_phys[l][k][i] * K_tensor[l][j] -
+                chris_phys[l][k][j] * K_tensor[i][l];
+        }
     }
 
+    // Use 'K-Theta' in CCZ4. Just 'K' in BSSN. Not a mistake, this is not to
+    // confuse with the typical 'K-2*Theta' that appears in the CCZ4 equations
+    data_t K_minus_theta = vars.K;
+    if (m_formulation == CCZ4RHS<>::USE_CCZ4)
+        K_minus_theta -= vars.Theta;
+
     // Calculate electric and magnetic fields
     FOR2(i, j)
     {
         out.E[i][j] = 0.0;
         out.B[i][j] = 0.0;
     }
 
-    FOR4(i, j, k, l)
-    {
-        out.B[i][j] +=
-            epsilon3_LUU[i][k][l] * covariant_deriv_K_tensor[l][j][k];
-    }
-
     FOR2(i, j)
     {
-        out.E[i][j] += ricci_and_Z_terms.LL[i][j] + vars.K * K_tensor[i][j];
-    }
+        out.E[i][j] +=
+            ricci_and_Z_terms.LL[i][j] + K_minus_theta * K_tensor[i][j];
 
-    FOR4(i, j, k, l)
-    {
-        out.E[i][j] += -K_tensor[i][k] * K_tensor[l][j] * h_UU[k][l] * vars.chi;
-    }
+        FOR2(k, l)
+        {
+            out.E[i][j] +=
+                -K_tensor[i][k] * K_tensor[l][j] * h_UU[k][l] * vars.chi;
 
-    if (m_formulation == CCZ4RHS<>::USE_CCZ4)
-    {
-        // In CCZ4 case do explicit symmetrization; BSSN case relies on momentum
-        // constraint satisfaction instead
-        TensorAlgebra::make_symmetric(out.B);
+            out.B[i][j] +=
+                epsilon3_LUU[i][k][l] * covariant_deriv_K_tensor[l][j][k];
+        }
+    }
 
-        FOR2(i, j) { out.E[i][j] += -vars.Theta * K_tensor[i][j]; }
+    // For CCZ4, explicit symmetrization appears naturally
+    // For BSSN, this relies on Momentum constraint satisfaction
+    // (see Alcubierre chapter 8.3, from eq. 8.3.15 onwards)
```

Comment:
This comment is confusing as you are referring to the relevant expressions and not to what is done in the code. Can you  make this more explicit (or say that we also explicitly symmetrize in the BSSN case even though the expressions aren't that way).